docs/README.md

```diff
@@ -33,14 +33,14 @@ pip install git+https://github.com/huggingface/doc-builder
 **NOTE**
 
 You only need to generate the documentation to inspect it locally (if you're planning changes and want to
-check how they look like before committing for instance). You don't have to commit the built documentation.
+check how they look before committing for instance). You don't have to commit to the built documentation.
```

Comment:
Might be better to revert this here to avoid changing the meaning of it (`git commit` the generated docs versus commit to it).

```suggestion
check how they look before committing for instance). You don't have to `git commit` the built documentation.
```